app/views/devise/registrations/new.html.erb

```diff
@@ -70,12 +70,10 @@
           <div class="recaptcha-error"></div>
           <div class="recaptcha" id="captcha-widget"></div>
           <div class="actions text-center">
+              <%= link_to "Have an account?", new_user_session_path,  class: "btn btn-primary btn-lg" unless Rails.env.production?%>
```

Comment:
Earlier it was mentioned the same condition. So, I kept it as similar. If it is not required I can remove it. @prem548